devops/templates/limited-installation-job-template.yml

```diff
@@ -0,0 +1,46 @@
+parameters:
```

Comment:
it'd be nice to have a comment on top of these yml files and say what each one is supposed to be doing

Reply:
I could not agree more!

Reply:
Some of them do already, but I'll add descriptions where there are none yet